packages/markdown/loader.js

```diff
@@ -38,9 +38,9 @@ const packages = [
   {
     product: 'material-ui',
     paths: [
+      path.join(__dirname, '../../packages/mui-base/src'),
```

Comment:
Material UI API links should win over the Base UI ones in case of a conflict in names.